Themes/default/languages/LightPortal/.spanish_latin.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
 /**
- * Spanish translation by Rock Lee https://www.bombercode.net | Rev. 4.3
+ * .spanish latin language file Rev. 4.3
+ * Spanish translation by Rock Lee https://www.bombercode.net
  *
  * @package Light Portal
+ * @author Bugo https://dragomano.ru/mods/light-portal
```

Comment:
I already use a different type of comments in the main language files :)

Reply:
If I had several doubts like placing it in itself, that's why I put a general one and then changed it.